docs/concepts/gas-enabled-network.md

```diff
@@ -0,0 +1,83 @@
+---
+description: Gas enabled networks
+---
+
+# Gas enabled networks
+
+By default, GoQuorum is a [free gas network](free-gas-network.md), which means gas is not consumed by transactions.
+However, gas price can be enabled if required.
+
+Transactions use computational resources, so they have associated costs.
+*Gas* is the cost unit and *gas price* is the price per gas unit.
+The transaction cost is the gas used multiplied by the gas price.
+
+If gas is enabled, then similar to public Ethereum networks, the account submitting the transaction pays the transaction cost, in Ether.
+The miner (or validator, in proof of authority networks) that includes the transaction in a block receives the
+transaction cost.
+
+## Using gas in GoQuorum
+
+### Enabling gas price
+
+To enable gas price in GoQuorum, set `enableGasPriceBlock` in the GoQuorum
+[genesis file](../configure-and-manage/configure/genesis-file/genesis-options.md) `config` object to a future block, when the entire network is
+ready to transact with gas price enabled transactions.
+All GoQuorum nodes in the network must be initialized with the same `enableGasPriceBlock` value.
+
+!!! note
+
+    You need to re-run `geth init` when you update the genesis file to incorporate the changes. This does not delete or modify any current sync process or saved blocks.
+
+## Nuances of gas price in GoQuorum
+
+There are some nuances around gas price and the rewarded account, which vary depending on which consensus mechanism is selected.
+These will impact how a gas enabled GoQuorum network is set up, and are described below.
+
+### Private transactions
+
+The amount of gas consumed by a private transaction is only the `intrinsic gas` value, and this is based on the private transaction hash held in the `data` field (rather than the entire private payload).
+The actual cost of creating or executing a private contract cannot be used, as the minter might not be a party to the transaction, and hence unable to calculate the real cost.
```

Comment:
The actual cost of creating or executing a private contract cannot be used. Members who are not party to the transaction will be unable to calculate the private gas cost and the network will not be able to reach a consensus.